stats/eventhubs/config_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2016 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package eventhubs
+
+import (
+	"database/sql"
+	"testing"
+
+	"github.com/loadimpact/k6/lib/types"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/guregu/null.v3"
+)
+
+func TestNewConfig(t *testing.T) {
+	config := NewConfig()
+	assert.Equal(t, sql.NullString(sql.NullString{String: "", Valid: true}), config.ConnectionString.NullString)
```

Comment:
No need to import `sql` here, no?

```suggestion
	assert.Equal(t, null.NewString("", false), config.ConnectionString)
```